adr/ADR-8.md

```diff
@@ -309,7 +310,11 @@ A bucket is a Stream with these properties:
    * The minimum allowed size is 1. When creating a stream, 1 should be used when the user does not supply a value.
  * Safe key purges that deletes history requires rollup to be enabled for the stream using `rollup_hdrs`
  * Write replicas are File backed and can have a varying R value
- * Overall Key TTL is managed using the `max_age` key
+* Overall Key TTL is managed using the `max_age` key
```

Comment:
Formatting is out of line with the rest here - think you need a space infant

Reply:
fixed